internal/dag/policy.go

```diff
@@ -665,16 +665,20 @@ func loadBalancerRequestHashPolicies(lbp *contour_api_v1.LoadBalancerPolicy, val
 		hashSourceIPSet := false
 		// Map of unique header names.
 		headerHashPolicies := map[string]bool{}
+		// Map of unique query parameter names.
+		queryParameterHashPolicies := map[string]bool{}
```

Comment:
nit: could make this and `headerHashPolicies` a `map[string]struct{}{}` so the value takes up no space, don't know why i didn't do that to start with